db/schema.rb

```diff
@@ -45,8 +45,11 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "guest", default: false
-    t.index ["email"], name: "index_users_on_email", unique: true
```

Comment:
@tpendragon I believe that recreating the staging db will fix the unique guest user issue because of this change